src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -106,42 +89,6 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
             return await EditAsync(user, context);
         }
 
-        var isEditingDisabled = IsCurrentUser(user);
-
-        if (!isEditingDisabled && !model.IsEnabled && user.IsEnabled)
-        {
-            var enabledUsersOfAdminRole = (await _userManager.GetUsersInRoleAsync(OrchardCoreConstants.Roles.Administrator))
-                .Cast<User>()
-                .Where(user => user.IsEnabled)
-                .ToList();
-
-            if (enabledUsersOfAdminRole.Count == 1 && user.UserId == enabledUsersOfAdminRole.First().UserId)
```

Comment:
You should handle this case as well to make sure we don't allow disabling the only admin.